build/Makefile

```diff
@@ -62,6 +62,13 @@ bundle:
 	$(DENO) run --allow-read --allow-write hack/bundle.js $(OUT_WA) $(OUT_BN)
 	$(DENO) fmt $(OUT_BN)
 
+size.txt: sqlite.js sqlite.wasm
```

Comment:
(Github won't let me comment on the non-changed line), but I think this would be valuable to add to the `release` target. That way, it will get run automatically after a release build, so the figures are always up to date and if someone submits a PR, it's likely the will have run it locally.

As part of this as well: some people might not have the brotli cli installed, so I think it should fail gracefully in those cases such that development won't be affected (the CI can then jump in and update the files in the automatic run)

(The second point would only make sense it we decide to add it to the release build though; what do you thin @jeremyBanks?)

Reply:
Good points, I agree. I will make the changes.